package.json

```diff
@@ -174,7 +174,7 @@
   },
   "dependencies": {
     "@brave/brave-ui": "0.40.6",
-    "@brave/leo": "github:brave/leo#63e98a1bb14cf87dcb8cbd4436e9e38615840447",
+    "@brave/leo": "github:brave/leo#skia-colors",
```

Comment:
this is not the final form, right? I expect to have a commit here.

Reply:
Yup, I mentioned this in the PR description - I don't want to merge the `skia-colors` branch into Nala until we're in agreement about what shape this is going to be in `brave-core` because it'll block bug fixes.